pkgs/development/python-modules/vdf/default.nix

```diff
@@ -13,17 +15,30 @@ buildPythonPackage rec {
 
   src = fetchFromGitHub {
     owner = "ValvePython";
-    repo = pname;
-    rev = "v${version}";
+    repo = "vdf";
+    rev = "refs/tags/v${version}";
     hash = "sha256-6ozglzZZNKDtADkHwxX2Zsnkh6BE8WbcRcC9HkTTgPU=";
   };
 
+  patches = [
+    # Support appinfo.vdf v29 (required by protontricks 1.12.0)
+    (fetchpatch2 {
+      url = "https://github.com/Matoking/vdf/commit/981cad270c2558aeb8eccaf42cfcf9fabbbed199.patch";
+      hash = "sha256-kLAbbB0WHjxq4rokLoGTPx43BU44EshteR59Ey9JnXo=";
+    })
+  ];
+
   nativeCheckInputs = [
     mock
     pytestCheckHook
   ];
+
   pythonImportsCheck = [ "vdf" ];
 
+  # The default python updater doesn't work because pygls doesn't use
+  # GitHub releases, only tags
```

Comment:
That comment needs an update

Reply:
How so? I just added it in this PR.

Reply:
Oh wow sorry, I just realized that I forgot to update `pygls` -> `vdf` when I copied it over from a comment I had in `pygls`. 

It's been removed in `pygls` since https://github.com/kira-bruneau/nur-packages/commit/d81992809fc9b5db932bda72e457af2acf062361#diff-6d296d0fb813abdd80dbe3349fc25d7a0afbb784bf97a50f6228e1af4e6e0b29L57-L60.

I updated the comment to this instead:
```nix
  # Use nix-update-script instead of the default python updater
  # The python updater requires GitHub releases, but vdf only uses tags
```